Documentation/ceph-upgrade.md

```diff
@@ -89,6 +89,40 @@ you should change to match where your images are located.
         value: "quay.io/k8scsi/csi-resizer:v0.4.0"
 ```
 
+### Retroactively enabling volume expansion
+
+When upgrading to the CSI 2.0 driver with existing persistent volumes, the following considerations should be taken into account when choosing to leverage the volume expansion feature:
+
+#### Remediate existing storage classes
+
+It is also necessary to ensure that the controlling StorageClass have the required changes
+to support Volume Expansion see the documentation about configuring [dynamically expanding volumes](https://rook.io/docs/rook/master/ceph-csi-drivers.html#dynamically-expand-volume).
+
+#### Remediate existing persistent volumes
+
+For any existing persistent volumes that you wish to 'upgrade' to allow expansion, you must
+patch them with some needed metadata.
+
+You can do this by:
+
+1. create a patch file, e.g. `csi-expansion-patch.yaml` with the following:
+
+```yaml
+spec:
+  csi:
+    controllerExpandSecretRef:
```

Comment:
Does this attribute to be added to the storage class so new PVs will get it? Or will new PVs get it automatically?

Reply:
we need  to ask the user to recreate the storage class, so new PVC can also make use of it

Reply:
Yeah, we need to also patch/edit the necessary StorageClass to add the `csi.storage.k8s.io/controller-expand-secret-name` & `csi.storage.k8s.io/controller-expand-secret-namespace` in addition to the `allowVolumeExpansion: true`.

As I understand it, this will influence all net-new PVs but existing PVs will need to be edited/patched in order to work with expansion.